resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -64,3 +64,24 @@ function createListElement(date, type, text, id) {
 
   return liElement;
 }
+
+/**
+ * Fetches the blobstore-serve to sends its response as an array buffer to the Adobe DC View
+ */
+async function getRevieweeResume() {
+  fetch('/blobstore-serve')
+    .then((response) => {
+      var adobeDCView = new AdobeDC.View({
+        clientId: "b98bbf69d44442479396583253ac267c",
+        divId: "adobe-dc-view"
+      });
+      adobeDCView.previewFile({
```

Comment:
You can also play with supplying an embed mode to change the PDF viewer's appearance. See https://www.adobe.com/devnet-docs/dcsdk_io/viewSDK/howtos.html#embed-modes